gemfiles/Gemfile.rubocop-old

```diff
@@ -3,3 +3,4 @@
 eval_gemfile "../Gemfile"
 
 gem "rubocop", "< 0.87"
+gem "rubocop-ast", "< 0.4.0"
```

Comment:
I was looking at this and thinking "this should not be here", but rubocop breaks without it. _rubocop_. `erb_lint`'s test suite works just fine.

That's just awful. It looks like `rubocop 0.86.0` is not compatible with newer `rubocop-ast` versions but doesn't specify a corresponding version restriction.

I wonder if we can easily still run `erb_lint`'s test suite against unrestricted versions of `rubocop-ast` and fix this problem differently.

Reply:
I'm going to try and set up rubocop as a separate build step

Reply:
Let's see if this works https://travis-ci.org/github/Shopify/erb-lint/builds/748234203

Travis is backed up pretty badly, so it looks like this will have to wait until tomorrow.

Reply:
Yeah, this works. If you want, cherry-pick my three commits from https://github.com/splitwise/erb-lint/compare/jess/remove-version-pinning...Shopify:test-new-rubocop.

Reply:
That would make this mergeable in my eyes

Reply:
Thanks for the additional work to clean up the tests.  I've updated my branch.